newrelic-alerts-configurator/src/main/java/com/ocadotechnology/newrelic/alertsconfigurator/configuration/condition/signal/SignalFillOption.java

```diff
@@ -0,0 +1,15 @@
+package com.ocadotechnology.newrelic.alertsconfigurator.configuration.condition.signal;
+
+public enum SignalFillOption {
+    NONE("none"), LAST_KNOWN_VALUE("last_value");
+
+    private String value;
+
+    SignalFillOption(String value) {
+        this.value = value;
+    }
+
+    public String getValue() {
+        return value;
+    }
```

Comment:
In most enums (although I can see that not everywhere) we just use lombok:
```
@Getter
@AllArgsConstructor
```

Reply:
OK